opentelemetry-api/src/opentelemetry/util/_importlib_metadata.py

```diff
@@ -14,7 +14,7 @@
 
 # FIXME: Use importlib.metadata when support for 3.11 is dropped if the rest of
 # the supported versions at that time have the same API.
-from importlib_metadata import (  # type: ignore
+from importlib.metadata import (
```

Comment:
```suggestion
from importlib_metadata import (
```

Nope, you cannot change implementation